source/overview.rst

```diff
@@ -42,7 +42,7 @@ Applications
 ============
 This section will list our applications and explain the motivations
 behind them, as well as their dependencies.
-This includes only repositories that are hosted on pcdshub with a
+This includes only repositories that are hosted on pcdshub and have a
```

Comment:
`This includes only repositories that are hosted on pcdshub` is actually a lie

This page is probably due for an update soon anyway

Reply:
Just those, but I suspect more have been added since the original write of this page

Reply:
Opened new issue #16